### PR TITLE
New National Park campus records

### PR DESCRIPTION
**Pull request by stepps00**
This PR adds new National Park campus records for Great Britain. I was able to find an open data source for these places:

https://en.wikipedia.org/wiki/List_of_national_parks_of_France

The second commit in this PR adds names from wikidata; each of these new records should have a Wikidata concordance and various name translations. PIP work was completed manually, can be merged once approved.

### CODE REVIEW

File: data/174/641/506/1/1746415061.geojson [hunk 1]
```diff
@@ -0,0 +1,154 @@
+{
+  "id": 1746415061,
+  "type": "Feature",
+  "properties": {
+    "date:inception_lower":"1970-01-01",
+    "date:inception_upper":"1970-12-31",
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"1970",
+    "eu-cdda:cddaCountr":"FR",
+    "eu-cdda:cddaId":660,
```
Comment (nvkelso) [inline data/174/641/506/1/1746415061.geojson:10]: Same comment as in the UK... if this ID is only unique for the layer not the provider than the source prefix needs to be more specific, please.

**Review by nvkelso — COMMENTED** [review]
The geometry precision is just a question... the unique ID in the concondances w/r/t the source name is something we should sort out.

File: data/174/641/506/1/1746415061.geojson (hunk 1 above)
Comment (stepps00) [reply data/174/641/506/1/1746415061.geojson:10]: This is a different source than the UK data, but these are unique codes to the whole CDDA database.. so I think we're good here?

Comment (nvkelso) [reply data/174/641/506/1/1746415061.geojson:10]: Agree we're good here for EU data. My remaining comment is just for UK.

**Comment by nvkelso** [comment]
We sorted out that this EU data source is specific just to the protected areas so clear to merge.

**Review by nvkelso — APPROVED** [review]

**Pull request merged**